DevCycle.SDK.Server.Common/Exception/DVCException.cs

```diff
@@ -24,5 +24,10 @@ public bool IsRetryable()
         {
             return (int)HttpStatusCode >= 500;
         }
+
+        public bool IsUserError()
```

Comment:
you should just use the existing `IsRetryable` method (above)

Reply:
do we care if we log errors under the 400 level?

Reply:
there arent errors under the 400 level

Reply:
oh right, cool - i'll change it